[2024-08-05T09:23:30.955+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-05T09:23:30.963+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.get_src_tables manual__2024-08-05T09:23:28.127297+00:00 [queued]>
[2024-08-05T09:23:30.966+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.get_src_tables manual__2024-08-05T09:23:28.127297+00:00 [queued]>
[2024-08-05T09:23:30.966+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-05T09:23:30.971+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_src_tables> on 2024-08-05 09:23:28.127297+00:00
[2024-08-05T09:23:30.975+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=3719) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-05T09:23:30.976+0000] {standard_task_runner.py:64} INFO - Started process 3730 to run task
[2024-08-05T09:23:30.977+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'get_src_tables', 'manual__2024-08-05T09:23:28.127297+00:00', '--job-id', '717', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpl24mnazo']
[2024-08-05T09:23:30.978+0000] {standard_task_runner.py:91} INFO - Job 717: Subtask get_src_tables
[2024-08-05T09:23:30.998+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.get_src_tables manual__2024-08-05T09:23:28.127297+00:00 [running]> on host b6b4a3e7bf9d
[2024-08-05T09:23:31.030+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='get_src_tables' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T09:23:28.127297+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-05T09:23:28.127297+00:00'
[2024-08-05T09:23:31.030+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-05T09:23:31.034+0000] {base.py:84} INFO - Using connection ID 'postgres_server_2' for task execution.
[2024-08-05T09:23:31.038+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:269: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  return psql.read_sql(sql, con=conn, params=parameters, **kwargs)

[2024-08-05T09:23:31.056+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-05T09:23:31.056+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-05T09:23:31.059+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=get_src_tables, run_id=manual__2024-08-05T09:23:28.127297+00:00, execution_date=20240805T092328, start_date=20240805T092330, end_date=20240805T092331
[2024-08-05T09:23:31.078+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-05T09:23:31.088+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-05T09:23:31.089+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
